app/views/rails_admin/main/_form_boolean.html.erb

```diff
@@ -1,7 +1,7 @@
 <% if field.nullable? %>
   <div class="btn-group" data-toggle="buttons">
     <% {'1': true, '0': false, '': nil}.each do |text, value| %>
-      <label class="<%= [field.css_classes[value], ("active" if field.form_value == value)] %> btn btn-default">
+      <label class="<%= [field.css_classes[value], ("active" if field.form_value == value)].compact.join(' ') %> btn btn-default">
```

Comment:
Could a test be added (or extended) to demonstrate the fix and would have caught the regression?

Reply:
I think the regression was caused when we decided to add the `nil` field and change the layout ( mainly with haml integration ).
I don't think there's a necessity to amend the tests as this was a UI bug ( not sure if there are UI tests )